packages/@apphosting/adapter-astro/src/interface.ts

```diff
@@ -0,0 +1,14 @@
+export interface AstroConfig {
```

Comment:
wonder if we can get this type directly from Astro?

Reply:
This Astro config is a modified version of the one supplied by Astro, we want the modified one so I think writing our own makes sense here.